packages/client/src/components/pages/muster-page/table-unit-muster-compliance.tsx

```diff
@@ -155,6 +119,57 @@ export const UnitMusterComplianceTable = (props: any) => {
   // Functions
   //
 
+  const getUnitComplianceData = async (organizationId: number, unitName: string, isoStartDate: string, isoEndDate: string): Promise<ComplianceData> => {
+    const xRange: string[] = [];
+    const yRange: number[] = [];
+    const colors: string[] = [];
+    let dayCount: number = 0;
+    let totalCompliance: number = 0;
+    const complianceStats: ComplianceStats = {
+      minCompliance: 100.0,
+      maxCompliance: 0,
+      avgCompliance: 0,
+    };
+    const res = await MusterClient.getMusterComplianceByDateRange(organizationId, unitName, { isoStartDate, isoEndDate });
```

Comment:
Probably `data` would be a clearer name for this variable, since it's not actually the response object.

Reply:
it's `result` lol, but yeah vague af!